### ISSUE TITLE
New Release date for changes related to laravel 5.4

### ISSUE THREAD

**Issue by codefun01** [moloquent/moloquent#87]
Hi,
There are many changes related to relationships and queue are done for Laravel 5.4.
When are you going to release those changes?

If not soon then what is the alternative to use in production as well as a development environment?

**Comment by amiri27** [comment]
Hi @darpit,

You mean Laravel 5.5?

Regards,

**Comment by darpit** [comment]
Nope.
Last release version is 4.0 and that was released in March.
When We can get new release ?

**Comment by codefun01** [comment]
Hi @amiri27 @pi0  ,

I am getting errors on a relationship (belongs to and has many) and also on database queue. I checked the source it seems these issues are resolved. Currently, i am using  0.4 version of moloquent.
I am using laravel 5.4.
Kindly provide me how can I get those changes?

Regards,

**Comment by codefun01** [comment]
Hi @pi0,
Can you please provide me the alpha release?

**Comment by j-ibarra** [comment]
yes mani changes on Laravel 5.5, [see this issue](https://github.com/moloquent/moloquent/issues/85)

**Comment by codefun01** [comment]
I am using laravel 5.4
